LICENSE

```diff
@@ -631,8 +631,8 @@ to attach them to the start of each source file to most effectively
 state the exclusion of warranty; and each file should have at least
 the "copyright" line and a pointer to where the full notice is found.
 
-    OTPClient - Simple GTK+ software to generate OTPs
-    Copyright (C) 2019 Paolo Stivanin
+    <one line to give the program's name and a brief idea of what it does.>
```

Comment:
You should keep the original text, just update the year

Reply:
This is just the part of 'How to Apply These Terms to Your New Programs' section of the GPL license, it is not meant to be edited. The whole GPL license is supposed to be distributed verbatim anyway.

Explanation here: https://www.gnu.org/licenses/gpl-faq.html#GPLOmitPreamble